PWGHF/D2H/TableProducer/dataCreatorDV0Reduced.cxx

```diff
@@ -0,0 +1,184 @@
+  Produces<aod::StraCollisions> lfReducedCollision;  // Defined in PWGLF/DataModel/LFStrangenessTables.h
+  Produces<aod::HfOrigColCounts> hfCollisionCounter; // Defined in PWGHF/D2H/DataModel/ReducedDataModel.h
+  Produces<aod::HfRedVzeros> hfCandV0;               // Defined in PWGHF/D2H/DataModel/ReducedDataModel.h
+  Produces<aod::HfRedD> hfCandD;                     // Defined in PWGHF/D2H/DataModel/ReducedDataModel.h
+
+  // selection
+  Configurable<int> selectionFlagD{"selectionFlagD", 7, "Selection Flag for D"};
+
+  HfHelper hfHelper;
+
+  // O2DatabasePDG service
+  Service<o2::framework::O2DatabasePDG> pdg;
+
+  bool isHfCandResoConfigFilled = false;
+
+  using TracksPidAll = soa::Join<aod::pidTPCFullEl, aod::pidTPCFullMu, aod::pidTPCFullPi, aod::pidTPCFullKa, aod::pidTPCFullPr,
+                                 aod::pidTOFFullEl, aod::pidTOFFullMu, aod::pidTOFFullPi, aod::pidTOFFullKa, aod::pidTOFFullPr>;
+  using TracksPIDWithSel = soa::Join<aod::TracksWCovDcaExtra, TracksPidAll, aod::TrackSelection>;
+  using TracksPIDWithSelAndMc = soa::Join<TracksPIDWithSel, aod::McTrackLabels>;
+  using CandsDFiltered = soa::Filtered<soa::Join<aod::HfCand3Prong, aod::HfSelDplusToPiKPi>>;
+
+  Filter filterSelectCandidates = (aod::hf_sel_candidate_dplus::isSelDplusToPiKPi >= selectionFlagD);
+
+  Preslice<CandsDFiltered> candsDPerCollision = aod::track_association::collisionId;
+  Preslice<aod::TrackAssoc> trackIndicesPerCollision = aod::track_association::collisionId;
+  Preslice<aod::V0Datas> candsV0PerCollision = aod::track_association::collisionId;
+
+  HistogramRegistry registry{"registry"};
+
+  void init(InitContext const&)
+  {
+    // histograms
+    constexpr int kNBinsEvents = kNEvent;
+    std::string labels[kNBinsEvents];
+    labels[Event::Processed] = "processed";
+    labels[Event::NoDV0Selected] = "without DV0 pairs";
+    labels[Event::DV0Selected] = "with DV0 pairs";
+    static const AxisSpec axisEvents = {kNBinsEvents, 0.5, kNBinsEvents + 0.5, ""};
+    registry.add("hEvents", "Events;;entries", HistType::kTH1F, {axisEvents});
+    for (int iBin = 0; iBin < kNBinsEvents; iBin++) {
+      registry.get<TH1>(HIST("hEvents"))->GetXaxis()->SetBinLabel(iBin + 1, labels[iBin].data());
+    }
+    registry.add("hMassDToPiKPi", "D^{#minus} candidates;inv. mass (p^{#minus} K^{#plus} #pi^{#minus}) (GeV/#it{c}^{2});entries", {HistType::kTH1F, {{100, 1.7, 2}}});
+    registry.add("hMassV0", "K0^{s} candidates;inv. mass (#pi^{#plus}#pi^{#minus}) (GeV/#it{c}^{2});entries", {HistType::kTH1F, {{100, 0.35, 0.65}}});
+    registry.add("hPtD", "D^{#minus} candidates;D^{#minus} candidate #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}});
+    registry.add("hPtV0", "V0 candidates;V0 candidate #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}});
+    registry.add("hMassDs1", "Ds1 candidates;Ds1 candidate #inv. mass (GeV/#it{c}^{2});entries", {HistType::kTH1F, {{100, 2., 3.}}});
```

Comment:
```suggestion
    registry.add("hMassDs1ToDstarK0S", "Ds1 candidates;Ds1 candidate #it{M}(D^{*+}K0S) (GeV/#it{c}^{2});entries", {HistType::kTH1F, {{100, 2., 3.}}});
```
We could also consider to put the delta mass M(D K0S) - M(D) (this is what we typically do for resonances)

Reply:
Ok I will use the mass difference